docs/debug.md

```diff
@@ -11,7 +11,7 @@ following command:
 ```
 $ sudo -s
 # ulimit -c unlimited
-# echo "/home/ubuntu/apps/hello-world.canonical/1.0.18/core.%e.%p.%h.%t" > /proc/sys/kernel/core_pattern
+# echo "/home/ubuntu/snap/hello-world.canonical/1.0.18/core.%e.%p.%h.%t" > /proc/sys/kernel/core_pattern
```

Comment:
oh, ic, `sudo tee` as well in the pipe and echo as a normal user ;-)

ulimit -c unlimited can be done as a user iirc.